src/selectors/url-state.js

```diff
@@ -251,6 +251,34 @@ export const getUrlPredictor: Selector<
   }
 );
 
+// This function extracts a profile name from the path of a file inside a zip.
+function getProfileNameFromZipPath(path: string): string {
```

Comment:
Hm, would you mind putting this function inside the app-logic directory instead? I guess it's more like a app-logic function and not a selector.

Reply:
I moved it to the existing file about zipped files in profile-logic/zip-files.js